src/gwas/src/gwas/plots/cli.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+import argparse
+import logging
+import multiprocessing as mp
+from pathlib import Path
+
+from tqdm import tqdm
+
+from gwas.plots.helpers import (
+    check_existing_files,
+    filter_rois,
+    generate_and_save_manhattan_plot,
+    save_dataframe_as_pickle,
+)
+from gwas.plots.worker import create_dataframe_all_chr
+
+
+def parse_args() -> argparse.Namespace:
+    """Parses command-line arguments"""
+    parser = argparse.ArgumentParser(description="Generate Manhattan & QQ Plots")
+    parser.add_argument(
+        "--metadata_directory",
+        help="Insert path to the metadata directory",
+        required=True,
+    )
+    parser.add_argument(
+        "--scoredata_directory",
+        help="Insert path to the scoredata directory",
+        required=True,
+    )
+    parser.add_argument(
+        "--csvroi_file",
+        help="Insert path to the csv rois file",
+        default=None,
+    )
+    parser.add_argument(
+        "output_directory",
```

Comment:
I would prefer if this would be named `--output-directory`, not a positional argument

Reply:
Updated

```
    parser.add_argument(
        "--output_directory",
        help="Insert path to output directory for dataframes and plots",
        required=True,
    )
```